go/cmd/vttestserver/vttestserver_test.go

```diff
@@ -308,6 +311,39 @@ func TestMtlsAuthUnauthorizedFails(t *testing.T) {
 	assert.Contains(t, err.Error(), "code = Unauthenticated desc = client certificate not authorized")
 }
 
+func TestExternalTopoServerConsul(t *testing.T) {
+	// Start a single consul in the background.
+	cmd, configFilename, serverAddr := test.StartConsul(t, "")
+	defer func() {
+		// Alerts command did not run successful
+		if err := cmd.Process.Kill(); err != nil {
```

Comment:
if we use `consul agent -dev` i think we could do `consul leave` for a graceful shutdown and don't need to call `os.Remove`

Reply:
tried `consul leave` and it adds 10 seconds for grace shutdown. keeping process.kill